build_tasks/thumbnails.js

```diff
@@ -93,9 +93,9 @@ const checkFolderExists = (path) => {
  * @param {array} filepaths - array of filepaths
  * @param {object} transform - Sharp parameters and dist folder
  */
-const makeThumbnails = async (filepaths, transform) => {
```

Comment:
async is not required as we don't have any "awaits" and the method is already returning an Promise.all